selenium_tests/ab_tool/page_objects/ab_add_experiment_page.py

```diff
@@ -0,0 +1,63 @@
+"""This is the page object of the Add Experiment Page"""
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from selenium_tests.ab_tool.page_objects.ab_base import ABBasePage
+from selenium.common.exceptions import NoSuchElementException
+
+
+### Locators for the Add_experiment page###
+class Locator2(object):
+    _experimentname = (By.ID, "experimentName") 
+    _notes_field = (By.ID, "experimentNotes")
+    _new_track_name = (By.ID,"newTrackName")
+    _add_tracks = (By.ID, "addTrack") 
+    _submit_button = (By.ID, "create-confirm")
+    _confirm_create = (By.ID, "createNow")
+    _uniform_checkbox = (By.ID, "uniformRandom")
+    _trash_icon = (By.CSS_SELECTOR, "i.fa.fa-trash")
+
+class AddExperimentPage(ABBasePage):
+    def __init__(self, driver):
+        super(AddExperimentPage, self).__init__(driver)  
```

Comment:
You can remove this entire function...if you're not doing anything local to the subclass, you don't need to implement `__init__()`...the superclass' `__init__()` will be called for you automatically.